tests/zkevm/test_proof_size.py

```diff
@@ -0,0 +1,126 @@
+"""
+A series of stress tests that assess the ability of a specific
+opcode or precompile to process large proofs in a block.
```

Comment:
opcodes/precompiles wouldn't process proofs. 

These tests to be added should be seen as regular blockchain tests that so happen to be edge cases that we want to test in a zkEVM. In that sense, they should not be any different than other blockchain tests

Reply:
Agreed. These tests are no different. By "process" I mean making having these opcode / precompile to take the code as input.